apps/server/src/mindmap/dto/create-mindmap.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { IsNotEmpty, IsOptional, IsString } from "class-validator";
+
+export class CreateMindmapDto {
+
+    @IsString()
+    @IsOptional()
+    name: string;
+
+    @IsString()
+    @IsOptional()
+    path: string;
```

Comment:
We do not need `path` here. `path` is created from backend. `path` will only be in UpdateMindmapDto.